bgpd/bgp_route.c

```diff
@@ -3186,6 +3187,24 @@ static void bgp_process_main_one(struct bgp *bgp, struct bgp_dest *dest,
 		bgp_unregister_for_label(dest);
 	}
 
+	if (safi == SAFI_MPLS_VPN && new_select && new_select->extra &&
+	    new_select->extra->bgp_orig) {
+		bgp_orig = new_select->extra->bgp_orig;
+		if (CHECK_FLAG(bgp_orig->vpn_policy[afi].flags,
+			       BGP_VPN_POLICY_TOVPN_LABEL_TO_SEND)) {
```

Comment:
Two if in a row is not needed

Reply:
I remove two if in a row. However, to ease the read, I keep two consecutive if so that bgp_orig can be used.